engine/src/main/java/org/terasology/persistence/internal/ReadWriteStorageManager.java

```diff
@@ -369,6 +371,14 @@ private void addGameManifestToSaveTransaction(SaveTransactionBuilder saveTransac
         for (WorldInfo worldInfo: worlds) {
             gameManifest.addWorld(worldInfo);
         }
+
+        WorldGenerator worldGenerator = CoreRegistry.get(WorldGenerator.class);
+        if (worldGenerator != null) {
+            WorldConfigurator worldConfigurator = worldGenerator.getConfigurator();
+            Map<String, Component> params = worldConfigurator.getProperties();
+            gameManifest.setModuleConfigs(worldGenerator.getUri(), params);
```

Comment:
❔  so this is doing a thing to make sure that the WorldConfigurator properties are current in the GameManifest when it is saved.

but how does GameManifest know how to save its new `moduleConfigs` field? I didn't see a change to a GameManifestSerializer or anything like that.

Reply:
its written to the worldEntityRef.